serializations.md

```diff
@@ -532,18 +537,93 @@ The payload is a serialized signed channel offchain transaction or it is empty.
 ]
 ```
 
-The payload is a serialized signed channel offchain transaction and can not be empty.
+The payload is a serialized signed channel off-chain transaction and can not be empty.
 
-#### Channel offchain transaction
 
-The channel offchain transaction is not included directly in the transaction tree but indirectly as payload of:
+#### Channel off-chain update
+
+Channel update rounds are described by various updates. They all form the
+`update()` type. If not specified something different, any of the addreses
```

Comment:
Where is `update()` type specified?

Reply:
This section exactly: all of those form the `update()` type: it could be either transfer, deposit, withdrawal, contract create or contract call.

Reply:
Alternative proposal:

> Channel update rounds are described by various updates, that are defined in this subsection. Each mention of type `update()` in the rest of this document is meant to be understood as referring to any one of these updates. ...